client/components/LearnerViewDirective.js

```diff
@@ -370,6 +379,13 @@ tie.directive('learnerView', [function() {
           ].join('\n')
         ];
 
+        var SECONDS_TO_MILLISECONDS = 1000;
+        // Default time interval in seconds that codes will be auto saved.
+        // Codes will be auto-saved every minute.
```

Comment:
Replace entire comment with: Default time interval, in seconds, after which code will be auto-saved.